tests/subsys/app_event_manager/src/events/name_style_events.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _NAME_STYLE_EVENTS_H_
+#define _NAME_STYLE_EVENTS_H_
+
+/**
+ * @brief Name style events
+ * @defgroup name_style_event Name style events
+ *
+ * The events here are prepared to check the sections sorting.
+ * We are sorting event listeners by the event name and we relay
+ * on the assumption that when everything is sorted alphabetically, with the right
+ * postfixes added to tag listener begin and end, no other event would jump in between
+ * other event range tags.
+ * We find this not to be true when somebody does not follow our naming pattern
+ * where the event always finishes with _event postfix.
```

Comment:
No - it would not, after sorting:
```
button_event_a
button_event_p10
button_event_z
button_sim_event_a
button_sim_event_p10
button_sim_event_z
```

For this issue to appear it have to be:
button_event and button_event_sim_event - looks silly, but is possible.

Reply:
You are right. Please disregard my previous comment.